.github/workflows/release.yml

```
@@ -269,32 +48,17 @@ jobs:
-      - name: Upload artifacts to action summary
-        uses: actions/upload-artifact@v3
-        if: github.event_name != 'release'
-        with:
-          name: dashmate
-          path: packages/dashmate/dist/**
+          OSX_KEYCHAIN: ${{ runner.temp }}/app-signing.keychain-db
+        run: "${GITHUB_WORKSPACE}/scripts/pack_dashmate.sh macos"
 
       - name: Notarize MacOS Release Build
-        if: runner.os == 'macOS'
         run: |
```

actionlint

shellcheck reported issue in this script: SC2156:warning:1:56: Injecting filenames is fragile and insecure. Use parameters

(shellcheck)